android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -160,9 +160,8 @@ private void loadWebView() {
     appUrlConfig = Config.getString("server.url");
     appAllowNavigationConfig = Config.getArray("server.allowNavigation");
 
-    String port = getPort();
-    String authority = "localhost" + ":" + port;
-    localUrl = "https://" + authority;
+    String authority = "app";
+    localUrl = "capacitor://" + authority + "/";
```

Comment:
Should we ditch authority completely and move to a fixed, configurable hostname? The possibility of clash seems rare considering it's local to the app, no?

Reply:
I used `app` as authority to be consistent with iOS, but we can make it configurable on both